tests/core/test_context.py

```diff
@@ -1367,3 +1368,65 @@ def test_plan_runs_audits_on_dev_previews(sushi_context: Context, capsys, caplog
     assert "'not_null' audit error:" in log
     assert "'at_least_one_non_blocking' audit error:" in log
     assert "Target environment updated successfully" in stdout
+
+
+@pytest.mark.slow
+def test_model_linting(tmp_path: pathlib.Path) -> None:
```

Comment:
Let's test custom rules as well.

Reply:
Let's also test Python models while we're at it, both Python-based and SQL-based.